### ISSUE TITLE
Training/Validation split : independent validation dataset ?? 

### ISSUE THREAD

**Issue by anthonygofin** [nianticlabs/monodepth2#443]
Hi,

Thank you for your incredible work. I am training a monocular model on a custom dataset (36,228 training items and 4,529 validation items). Video sequences were taken at 30 fps. I am running with frame_ids = 0 -3 3. I added a function to monitor both the training and validation loss to avoid overfitting.

I looked at eigen zhou split and I was wondering if validation data are independent of training data.
Let's suppose that we use frame_ids = 0 -1 1. If "_drive0001_sync **10** r_" is a validation item, then we need "_drive0001_sync **9** r_" and "_drive0001_sync **11** r_" items to compute the loss. However either "_drive0001_sync **11** r_" or "_drive0001_sync **9** r_" could be training data (target views or even source views).

I agree that input triplets are never the same in validation and training. Moreover, if "_drive0001_sync **8** r_" is a target view in training, frame "_drive0001_sync **9** r_" is a source view, then the pose model predicts T_8->9 (the relative pose of source image 8 with respect to the target image 9). In validation, as target image is "_drive0001_sync **10** r_", the pose model predicts T_9->10, which is not the same as in training. So I would say we are not spilling training information into the validation set... but I am not sure. May I ask your thoughts about this?

Thank you in advance.

**Comment by daniyar-niantic** [comment]
Hi @anthonygofin 
Good question. As far as I understand, this split was defined by Eigen et al (https://arxiv.org/abs/1406.2283), hence why it is often called "Eigen split". Monodepth2 reuses the same split for direct comparison.
